src/main/java/com/exactpro/th2/conn/MicroserviceMain.java

```diff
@@ -144,11 +144,23 @@ public static void main(String[] args) {
 
             MessageRouter<EventBatch> eventBatchRouter = factory.getEventBatchRouter();
 
-            String rootEventID = storeEvent(eventBatchRouter, Event.start().endTimestamp()
+            var rootEvent = Event.start().endTimestamp()
                     .name("Connectivity '" + configuration.getSessionAlias() + "' " + Instant.now())
-                    .type("Microservice")).getId();
-
-            IServiceListener serviceListener = new ServiceListener(directionToSequence, new IMessageToProtoConverter(), configuration.getSessionAlias(), processor, eventBatchRouter, rootEventID);
+                    .type("Microservice");
+            var errorEventsRoot = rootEvent.addSubEventWithSamePeriod()
+                    .name("Errors")
+                    .type("ConnectivityErrors");
+            var serviceEventsRoot = rootEvent.addSubEventWithSamePeriod()
+                    .name("ServiceEvents")
+                    .type("ConnectivityServiceEvents");
```

Comment:
Sorry, but now cradle has the limitation 'event cannot refer to an event in another batch'
We must store every event as a single